scripts/deploy/iks/test-fvt.sh

```diff
@@ -49,19 +48,22 @@ run_fvt() {
   export PATH=/root/go/bin/:$PATH
 
   export NAMESPACE=${SERVING_NS}
+  export NAMESPACESCOPEMODE=true
   ginkgo -v --progress --fail-fast -p fvt/predictor fvt/scaleToZero --timeout 40m > fvt.out
   cat fvt.out
-  RUN_STATUS=$(cat fvt.out | awk '{ print $1}' | grep PASS)
+
+  #if [[ $(grep "Test Suite Passed" fvt.out) ]]; then
+  #  export NAMESPACE="modelmesh-user"
+  #  ginkgo -v --progress --fail-fast -p fvt/predictor fvt/scaleToZero --timeout 40m > fvt.out
+  #  cat fvt.out
+  #  if [[ $(grep "Test Suite Passed" fvt.out) ]]; then
+  #    REV=0
+  #  fi
+  #fi
 
   if [[ $(grep "Test Suite Passed" fvt.out) ]]; then
-    export NAMESPACE="modelmesh-user"
-    ginkgo -v --progress --fail-fast -p fvt/predictor fvt/scaleToZero --timeout 40m > fvt.out
-    cat fvt.out
-    if [[ $(grep "Test Suite Passed" fvt.out) ]]; then
       REV=0
-    fi
   fi
-
```

Comment:
I don't completely understand the changes in this file, are they temporary?

Reply:
Yes, this is temporarily to make the test only for the controller namespace in the namespace scope mode, until we have the FVTs all figured out for CSRs in cluster scope mode.